docs/basic-features/data-fetching/get-static-props.md

```diff
@@ -130,6 +130,12 @@ One of the reasons for this restriction is that React needs to have all the requ
 
 Also, you must use export `getStaticProps` as a standalone function — it will **not** work if you add `getStaticProps` as a property of the page component.
 
+## Passing props
+
+The props returned by `getStaticProps` will be automatically passed through to the page by the default app.
+
+If however you've created a [custom app](/docs/advanced-features/custom-app), ensure you're passing the props manually from your app to the page component, otherwise the page props will be empty.
```

Comment:
This note seems more fitting in the custom-app document, what do you think about moving it there?

Reply:
I don't think it's a good idea actually. There is already a similar note in the custom app doc (it could be fleshed out a bit to be fair) but it didn't help me because I had no idea to even look there! There was no reason for me to suspect that the custom app was the culprit and that reading the custom app doc would helpful

I kept looking through the `getStaticProps`, `getStaticPaths`, and the SSR docs, ie. the places I expected to be relevant

So this small note in the expected place would have saved me heaps of time